### DISCUSSION TITLE
MAUI dotnet 7 - Want to prompt for PIN/Fingerprint onResume of the app

### DISCUSSION THREAD

**Discussion by smurfster** [smstuebe/xamarin-fingerprint#238] [Q&A]
Is there a known method of prompting to authenticate when switching to a different app then coming back to my app?

**Comment by mrwcjoughin** [comment]
@smurfster  Technically you could use deep linking to do this.
But it's not allowed my Apple outright.
You are not allowed to make your app dependant on another app in iOS or iPadOS.
I built a very secure MFA system for a client and when they tried to use it to login into a separate app they created Apple would not approve it at all - despite multiple appeals process
